tests/introspection/iOS/iOSApiProtocolTest.cs

```diff
@@ -410,6 +410,137 @@ protected override bool Skip (Type type, string protocolName)
 					return !TestRuntime.CheckXcodeVersion (8,0);
 				}
 				break;
+
+			case "CALayerDelegate": // UIView now conforms to CALayerDelegate in iOS 10
+				switch (type.Name) {
+				case "UISearchBar":
+				case "UISegmentedControl":
+				case "UITableView":
+				case "UITableViewCell":
+				case "UITextField":
+				case "UITextView":
+				case "UIToolbar":
+				case "UIView":
+				case "MKPinAnnotationView":
+				case "UIImageView":
+				case "PHLivePhotoView":
+				case "UIInputView":
+				case "UILabel":
+				case "UIActionSheet":
+				case "UIButton":
+				case "UICollectionView":
+				case "UINavigationBar":
+				case "UIControl":
+				case "UIPickerView":
+				case "UIPageControl":
+				case "MPVolumeView":
+				case "UIPopoverBackgroundView":
+				case "UIProgressView":
+				case "UIRefreshControl":
+				case "HKActivityRingView":
+				case "UIScrollView":
+				case "CAInterAppAudioSwitcherView":
+				case "CAInterAppAudioTransportView":
+				case "UISlider":
+				case "UIStackView":
+				case "SCNView":
+				case "UIStepper":
+				case "UISwitch":
+				case "UITabBar":
+				case "UITableViewHeaderFooterView":
+				case "GLKView":
+				case "SKView":
+				case "MKMapView":
+				case "MKAnnotationView":
+				case "PKAddPassButton":
+				case "PKPaymentButton":
+				case "UIActivityIndicatorView":
+				case "UICollectionReusableView":
+				case "UIWebView":
+				case "UICollectionViewCell":
+				case "UIWindow":
+				case "UIDatePicker":
+				case "UIVisualEffectView":
+				case "WKWebView":
+				case "ADBannerView":
+#if !__WATCHOS__
+					return !TestRuntime.CheckSystemAndSDKVersion (10, 0);
```

Comment:
there's a new `CheckXcodeVersion` method that's simpler, just supply `8,0` and it will find out the corresponding SDK versions.